### ISSUE TITLE
Carepackage too aggressive with reported data.

### ISSUE THREAD

**Issue by labrys** [mylar3/mylar3#1317]
**Describe the bug**
The care package generator is too aggressive with the information it harvests.  There is no need to get all environment variables.  I use an external script for post processing which stores api keys in environment variables, all of which get exported during care package generation.

**CarePackage**
On the config page, there is a button labelled "CarePackage". Pressing this button will download a zip file containing a redacted copy of your config, your database, a summary of your operating environment, and your logs. Upload that file here.

If you're uncomfortable with pasting your CarePackage, include a DEBUG log containing the error from disk. 

Do not paste from the GUI log.
Do not just paste a traceback/error.

**Environment (please complete the following information):**
 - OS: [e.g. Windows10]
 - Branch / commit: [e.g. Dev/38aksi1]

**Additional information**
Add any other information about the problem here.

**Comment by evilhero** [comment]
Not sure if it's too aggressive as we don't do anything outside of a ``systeminfo`` or ``uname -a``, that's really it. We all know storing env variables that are sensitive isn't the best way to do things as they're easily accessible by any application, so I won't go down that road. However, we also don't really need to usually see environmental variables in regards to troubleshooting Mylar.

We could parse out the extracted data for environmental variables so that it's not included in the carepackage data - we do it for a few windows ones that are excessively verbose and not necessary. So adding in an exclusion for environmental vars would be easy enough I think. 

Can look into it more when time is available, or if someone beats me to the punch.

**Comment by labrys** [comment]
@evilhero https://12factor.net/config
Credentials in environment variables are a well-known and accepted place to store them.

**Comment by evilhero** [comment]
I mean, posting a link to a site that I've never heard of doesn't help. Anyone can make a site that says the exact opposite and I can post it. Here's one that proves my point: https://snyk.io/blog/keeping-docker-secrets-secure/

My initial point for sensitive data, env variables aren't secure as they're readily accessible by any application on the system. All Mylar does is a basic ``uname`` command on nix (``systeminfo`` on Windows), both of which can be run by non-admin users. If Mylar does a basic command and get can get that info, what about the apps that are actually trying to harvest stuff?

But this is getting off track, we'll look into parsing out env variables from the carepackage, but it's not a high priority item and free time isn't being kind atm - so the parsing out will get attended to when there's time available outside of possibly other issues.

**Comment by labrys** [comment]
I agree it's off topic.  In my case I encrypt the env vars with credentials, but the reason for this post is the next bug I'm writing up is directly related to environment variables.
